metagraph/src/graph/alignment/alignment.cpp

```diff
@@ -1164,17 +1164,17 @@ void Alignment::insert_gap_prefix(ssize_t gap_length,
     offset_ = graph.get_k() - 1;
 }
 
-
-bool spell_path(const DeBruijnGraph &graph,
-                const std::vector<DeBruijnGraph::node_index> &path,
-                std::string &seq,
-                size_t offset) {
+// Return the string spelled by the path. This path may have disconnects (if it came)
+// from a chain alignment), so this method inserts
```

Comment:
throws?

Reply:
I forgot to complete the comment. Fixed now.